packages/tools/src/tools/annotation/BidirectionalTool.ts

```diff
@@ -240,7 +240,7 @@ class BidirectionalTool extends AnnotationTool {
     } = this.hydrateBase<BidirectionalTool>(
       BidirectionalTool,
       enabledElement,
-      [],
+      axis[0],
```

Comment:
axis is an array of array of Point3, so axis[0] is an array of Point, so this is correct.  Will approve retroactively.